docs/hosting.md

```diff
@@ -138,22 +138,27 @@ python serve.py
 ## Setting up Mol* Viewer
 
 - To view the data, a [Volumes and Segmentations extension](https://github.com/molstar/molstar/tree/master/src/extensions/volumes-and-segmentations) is available as part of the [Mol* Viewer](https://github.com/molstar/molstar). 
-- Please refer to [Mol* Docs](https://molstar.org/docs/) for installing the plugin.
-- You can provide the URL to your server instance as `volumesAndSegmentationsDefaultServer` option when creating the [Viewer instance](https://molstar.org/docs/plugin/#viewer-wrapper), e.g.
-```ts
-molstar.Viewer.create('app', {
-    // other options
-    volumesAndSegmentationsDefaultServer: 'https://my.org/volumes-and-segmentations/v2'
-}).then(viewer => { 
-    // ...
-})
+- In order to install the plugin, run the following commands:
 ```
+git clone https://github.com/molstar/molstar.git
+cd molstar
+npm install
+npm run build
+```
+
+- Download [Embedded Volseg HTML file](./embedded_volseg.html) to `build/viewer` directory in your local copy of molstar repository
```

Comment:
- Maybe we can put the HTML  file content inline here and say to get `molstar.js` and `molstar.css` from that folder?
- Make a comment about the `                volumesAndSegmentationsDefaultServer: 'http://localhost:9000/v2',` line explaining it points to the server instance